app/src/main/java/com/example/hoanganh/tablayout/Fragment/FragmentB.java

```diff
@@ -0,0 +1,27 @@
+package com.example.hoanganh.tablayout.Fragment;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.hoanganh.tablayout.R;
+
+/**
+ * Created by hoanganh on 3/5/2019.
+ */
+
+public class FragmentB extends Fragment
```

Comment:
```suggestion
public class BFragment extends Fragment
```
Đặt lại tên cho có ý nghĩa